packages/playground/src/domains/agents/components/agent-view-transition.css

```
@@ -0,0 +1,141 @@
+/*
+ * Fallback for browsers without the View Transitions API (applied only there;
+ * see pages/agents/agent/index.tsx). Opacity-only: an in-flow translate would
+ * overflow the main panel and flash its scrollbar during the animation.
+ */
+@keyframes agent-view-enter {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
```

Stylelint (17.13.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)